bin/start-local.js

```diff
@@ -2,20 +2,31 @@
 var spawn = require('child_process').spawn,
     stdout = '',
     execJob,
+    os = require('os'),
     path = require('path'),
-    env = {cwd: path.join(__dirname, '..'), NODE_ENV: process.env.NODE_ENV},
+    yaml = require('js-yaml'),
+    fs = require('fs'),
+    env = {cwd: path.join(__dirname, '..'), NODE_ENV: process.env.NODE_ENV, PATH: process.env.PATH},
     workerJob = null,
     gmeConfig = require(__dirname + '/../config');
 
+const DEEPFORGE_CONDA_ENV =
+    yaml.safeLoad(fs.readFileSync(path.join(__dirname, '..', 'base-environment.yml'))).name;
+const DEEPFORGE_SERVER_COMMAND =
+    `${os.type() === 'Windows_NT' ? 'conda': 'source'} activate ${DEEPFORGE_CONDA_ENV} && node`;
+
 // Set the cache to the blob
-if (gmeConfig.blob.type === 'FS') {
+if  (gmeConfig.blob.type === 'FS') {
     process.env.DEEPFORGE_WORKER_CACHE = path.resolve(gmeConfig.blob.fsDir + '/wg-content');
 }
 
 // process.env.NODE_ENV = 'local';
-execJob = spawn('node', [
+execJob = spawn(DEEPFORGE_SERVER_COMMAND, [
```

Comment:
It is worth mentioning that this is only run when `deepforge start` is run without any arguments. Running `deepforge start -s` (starting only the server will not call `start-local.js`). This should probably be updated as it is only for legacy reasons. Unfortunately, this means that conda is only used when starting all the components at once and not when starting only the server.

Reply:
This has been addressed in https://github.com/deepforge-dev/deepforge/pull/1467#issuecomment-597306352